### ISSUE TITLE
Not deservedly leaving the game

### ISSUE THREAD

**Issue by Whoagir** [ValveSoftware/Dota2-Gameplay#5382]
### Description

We were trapped on the base, I couldn't get exps, but I went to set up wards and raised bounty runes. They gave me a liv -20 mm and lowered my decency, please give me back all this, I try every game, 70 minutes of my life for liv which was not? match id 6860856052

### Example Match ID (and possibly Timestamp)

65 - 70 min

### Screenshots

![image](https://user-images.githubusercontent.com/100317552/201501835-8199c5c1-c29a-4796-950f-fe00106dcc18.png)

**Comment by commentator24** [comment]
You will get abandon if you can't get xp within 5 min

**Comment by Whoagir** [comment]
Yes, but I played this game. I set wards, raised bounties, If we couldn't get out of the base, the anti-mage split, and other heroes killed creeps without even reaching our base, how could I get an exp so as not to die

**Comment by commentator24** [comment]
i understand v hard to get out of base, but I dont believe that not 1 creep reach your base within 5 min.
need to be within xp range when cores clear creeps. play around them as support.
or even smoke and go to jungle to stun once.
this 5 min xp thing is not a bug unfortunately.
